packages/components/src/custom-select-control-v2/custom-select.tsx

```diff
@@ -138,7 +142,13 @@ function _CustomSelect(
 					store={ store }
 					// Match legacy behavior (move selection rather than open the popover)
 					showOnKeyDown={ ! isLegacy }
+					aria-describedby={ describedBy ? describedByID : undefined }
 				/>
+				{ describedBy && (
+					<div id={ describedByID } hidden>
+						{ describedBy }
```

Comment:
I would like this prop to be renamed because it's not a 'describedBy`. Actually, it's a description i.e. the actual text for the description. For an example of how to rename, use an 'alias', and deprecate the old name, see https://github.com/WordPress/gutenberg/pull/63486